README.md

```diff
@@ -165,23 +185,23 @@ The `condition` flag signals to TigerBeetle that a 256-bit cryptographic conditi
   flags |= TransferFlags.condition
 ```
 
```

Comment:
I think the README should include an explanation of the `ledger` field, i.e. how you can't transfer money between accounts with different `ledger`s (and why that is a useful feature).

Reply:
Done.